src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -216,125 +221,130 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
-          <Spacer x={1} />
-          <Button
-            disabled={!(selectedAddressKey && selectedAddressKey.length > 0)}
-            onClick={() => {
-              if (selectedAddressKey) {
+  return (
+    <ModalDialog
+      width={["100%", 590]}
+      height={["100vh", "auto"]}
+      m={[0, 2]}
+      title={modalTitles[modalType]}
+      onClose={handleClose}
+    >
+      {modalType === ModalType.SUGGESTIONS ? (
+        <>
+          <Text>
+            To ensure prompt and accurate delivery, we suggest a modified
+            shipping address.
+          </Text>
+          <Spacer y={2} />
+          <RadioGroup
+            defaultValue={selectedAddressKey || ""}
+            onSelect={(selected: AddressOption["key"]) => {
+              setSelectedAddressKey(selected)
+            }}
+          >
+            {addressOptions.map(
+              ({ key, recommended, ...addressOption }) =>
+                addressOption.lines && (
+                  <BorderedRadio
+                    key={key}
+                    value={key}
+                    label={
+                      recommended
+                        ? AddressSuggestionRadioButton.recommended
+                        : AddressSuggestionRadioButton.user_address
+                    }
+                  >
+                    <Flex flexDirection="column">
+                      {addressOption.lines.map((line: string) => (
+                        <Text variant="xs" key={line}>
+                          {line}
+                        </Text>
+                      ))}
+                    </Flex>
+                  </BorderedRadio>
+                )
+            )}
+          </RadioGroup>
+          <Spacer y={4} />
+          <Flex width="100%" justifyContent="space-between">
+            <Button
+              onClick={handleBackToEdit}
+              variant="secondaryBlack"
+              flex={1}
+            >
+              Back to Edit
+            </Button>
+            <Spacer x={1} />
+            <Button
+              disabled={!(selectedAddressKey && selectedAddressKey.length > 0)}
+              onClick={() => {
+                if (selectedAddressKey) {
+                  trackEvent({
+                    action_type: "clickedValidationAddress",
+                    context_module: ContextModule.ordersShipping,
+                    context_page_owner_type: OwnerType.ordersShipping,
+                    context_page_owner_id: contextPageOwnerSlug,
+                    user_id: user?.id,
+                    subject: "Confirm your delivery address",
+                    option: selectedAddressKey.includes("suggestedAddress")
+                      ? "Recommended"
+                      : "What you entered",
+                    label: "Use This Address",
+                  })
+                }
+                chooseAddress()
+              }}
+              flex={1}
+            >
+              Use This Address
+            </Button>
```

Comment:
Consider moving these buttons to the `footer` slot in `ModalDialog` they look weird floating on mobile

Reply:
yeah, i considered that but maybe didn't because testing it on desktop with a super narrow view they also looked weird. But people don't use desktop browsers that way, will update

Reply:
update: starting on this it begins to suggest a bigger refactor that I'd like to tackle after analytics tests are in place, which I'm going to do next. Can we merge this and I will follow up this week?

Reply:
sure, non-blocking!